### ISSUE TITLE
New "Keyboard" shortcuts tab in Settings papercuts

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#20625]
### Your use case

![image](https://user-images.githubusercontent.com/2803622/149977550-be2e374b-c7ca-48d1-ac06-a8fb4877dbf4.png)

first of all, nice addition to have it in such an obvious place

- i hope the layout will get overhauled to more of a list, like the Preferences tab
- especially scrolling horizontally is bad
- "Search (must be enabled)" -> link to the setting
- customizable keybinds? :eyes: https://github.com/vector-im/element-web/issues/15076
- make this and all settings searchable https://github.com/vector-im/element-web/issues/9592
- there is a white corner in the lower right with dark mode

**Comment by SimonBrandner** [comment]
Will by mostly fixed by https://github.com/matrix-org/matrix-react-sdk/pull/7562

**Comment by HarHarLinks** [comment]
excellent, the PR addresses the first 2 points and should also be able to incorporate the 3rd and 6th.

customizable keybinds probably need their own issue then.

searchable settings might be part of ongoing IA things, and should be put on the appropriate list.

**Comment by SimonBrandner** [comment]
> excellent, the PR addresses the first 2 points and should also be able to incorporate the 3rd and 6th.
> 
> customizable keybinds probably need their own issue then.
> 
> searchable settings might be part of ongoing IA things, and should be put on the appropriate list.

Customizable key binds are a separate issue and so are searchable settings. My PR will hopefully address the rest. Some more play with customizable key binds might or might not come after that :P

**Comment by dbkr** [comment]
It would be good if we could split these out into separate issues and have this as a meta-issue. That or at least turn the items into checkboxes (although some of them are huge, so I don't really think this is appropriate). Let's revisit this once the above PR is merged & see what's left.

**Comment by SimonBrandner** [comment]
+ [x] i hope the layout will get overhauled to more of a list, like the Preferences tab
+ [x] especially scrolling horizontally is bad
+ [ ] "Search (must be enabled)" -> link to the setting
+ [ ] customizable keybinds? eyes The ability to set custom keybinds. #15076
+ [ ] make this and all settings searchable Add a way to search settings #9592
+ [x] there is a white corner in the lower right with dark mode

**Comment by SimonBrandner** [comment]
Closing in favour of of the open issues
